lib/matplotlib/colors.py

```diff
@@ -184,13 +184,19 @@ def to_rgba(c, alpha=None):
     c : Matplotlib color or ``np.ma.masked``
 
     alpha : float, optional
-        If *alpha* is not ``None``, it forces the alpha value, except if *c* is
-        ``"none"`` (case-insensitive), which always maps to ``(0, 0, 0, 0)``.
+        If given, force the alpha value of the returned RGBA tuple to *alpha*.
+
+        If None, the alpha value from *c* is used, if it has alpha information,
```

Comment:
```suggestion
        If alpha is None, then the alpha value from *c* is used. If *c* does not have an alpha channel,  then alpha defaults to 1.
```
not sure if this is better, but wanted to unpack the nesting a bit.